sections/design/why-react.tex

```diff
@@ -0,0 +1,23 @@
+\section{Why React?}
+For our frontend, we considered two frameworks and a library: Vue.js, Angular and React.
+All of these web development tools are popular according to npm trends as well as the StackOverflow survey and are used by many different companies\cite{VueReactAngular}.
+\\
+Each of these web development tools have extended documentation and tutorials to get started on each their websites.
+\\\\
+Angular is the oldest of the three options we looked into and is created and maintained by Google\cite{AngularHistory}.
+Performance wise Angular should be the slowest of the three and also the one that takes up the most space\cite{VueReactAngular}.
+This is because Angular also provides templates and testing utilities for its user.
+It is a complete package and should provide the developer with all the functionality he needs for a frontend project.
+\\\\
+Vue.js is the newest of the three it was developed by an ex-engineer of Google and is the one that uses the least amount of space, making it good for small web projects\cite{ComparisonWithOtherFrameworks}.
+Vue.js, like Angular, is also a framework and is designed around its ecosystem which offers its developers add-ons like Vue.js Router which makes it a bit more flexible than Angular since it does not come in one big package.
+\\\\
+React, unlike the two other, is only a library\cite{ReactJS}, not a lot is provided in the React library which in turn gives the developer the option to choose what programming tools he or she wants to use in their project.
+It is developed and maintained by Facebook\cite{ReactHistory}.
+Space wise it only uses a bit more than Vue.js which still makes it a good choice for small projects.
+\\\\
+Most of the members from the group have already used Angular for a previous project and wanted to try one of the other options, ruling out Angular. 
+\\
+The decision between Vue.js and React was difficult but the extra flexibility of react spiked the most interest from the team even though Vue.js should be easier to learn according to the internet\cite{ComparisonWithOtherFrameworks}.
+According to Google trends React is also much more frequently searched for compared to vue.js which lead us to think it is easier to find documentation and examples.
```

Comment:
```suggestion
According to Google trends React is also much more frequently searched for compared to vue.js which led us to think it would be easier to find documentation and examples.
```